### ISSUE TITLE
Update fails if the app is automatically launched after installation

### ISSUE THREAD

**Issue by zachary-jomedia** [sparkle-project/Sparkle#410]
I try to update my app from appv1 to appv2. When I install appv1.pkg, appv1.app is automatically launched from inside of a script after installation and I see the Sparkle update window. If I click on Install Update, the update fails. Here is the log file.

2014-08-18 15:41:34 +0000: ===== mygames-launcher =====
2014-08-18 15:41:59 +0000: Extracting /Users/mac1/Library/Application Support/my games/my games 1.6 3/mygames-update.zip using 'ditto -x -k - "$DESTINATION"'
2014-08-18 15:42:04 +0000: releaseFromQuarantine
2014-08-18 15:42:04 +0000: chown -R 0:80 /Users/mac1/Library/Application Support/my games/my games 1.6 3/mygameslauncher.app failed.

If I click on Remind Me Later when I see Sparkle update window, close the app and relaunch it, the update works fine.

Then I stopped automatic launch of appv1.app after installation of appv1.pkg. Instead, I launch appv1.app from Terminal or Finder, the update works fine. Does anybody have any idea what is going on here? Thanks!

**Comment by Codengineer** [comment]
I am having the same issue. How was this solved?

**Comment by pornel** [comment]
The solution is not to launch the app from the installer. Leave it to Sparkle to handle all (re)launching.

**Comment by zachary-jomedia** [comment]
I was unable to solve the issue. I remembered I followed their instruction
to get around the issue.

Thanks!
Zachary

**Comment by Codengineer** [comment]
I found a solution.  I'm using Packages to build my pkg installer and I have an Apple Script  FirstTimeLaunch.scpt that launches the application as soon as the Installer quits. I found out that this Apple Script was ran with root privileges. Changing the code to run the script as the current user somehow fixed the issue:
`sudo -n -u $USER osascript FirstTimeLaunch.scpt $DSTROOT &`

Thanks for the original post that put me in the right direction and saved me hours of frustration.
